function.py

```diff
@@ -0,0 +1,6 @@
+
+def function(n):
```

Comment:
renaming the function and parameter can help understand the purpose of this function